src/gdal/rasterband.jl

```diff
@@ -257,8 +257,8 @@ _getrasterscale(hBand::GDALRasterBandH, pbSuccess::Ptr{Cint}) =
     GDALGetRasterScale(hBand, pbSuccess)::Cdouble
 
 function getrasterscale(rasterband::GDALRasterBandH)
-    success = Ref{Cint}()
```

Comment:
I updated according to your comment.  However, that entailed to also update `src/clang/gdal_cfuns.jl` which I believe is a auto-generated file.  So I'm not sure this is a better band-aid.  I left the previous version as a commit, so I can revert back.  Let me know what's better.